bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/syntax/sp/Instrument.scala

```diff
@@ -0,0 +1,35 @@
+package edu.gemini.spModel.syntax.sp
+
+import edu.gemini.pot.sp.Instrument
+import edu.gemini.pot.sp.Instrument._
+
+// NB: it would make sense to add this to the Instrument enum itself, but that
+// would break serialization.
+
+final class InstrumentOps(val self: Instrument) extends AnyVal {
+
+  /**
+   * Is this a "visitor" instrument.  In other words, is it fully integrated
+   * with the observing system and DHS?
+   */
+  def isVisitor: Boolean =
+    self match {
+      case AcquisitionCamera | Phoenix | Texes | Visitor => true
```

Comment:
I guess we don't have Zorro, Alopeke, DSSI right? They are just Visitors

Reply:
Right, there is no specific component for those.